lib/event_date.php

```diff
@@ -179,7 +179,7 @@ public function geTeaser() : ?string
         return $this->getValue("teaser");
     }
     
-    public function getPrice()
+    public function getPrice(): string
     {
         $offer = rex_yform_manager_table::get('rex_event_date_offer')->query()->where("date_id", $this->getValue('id'))->find();
 
```

Comment:
```suggestion
    public function getPrice(): ?event_offer
    {
        $offer = rex_yform_manager_table::get('rex_event_date_offer')->query()->where("date_id", $this->getValue('id'))->find();

```

Bin mir hier nicht sicher, am besten nochmal nachsehen. Es kommt ja ein Dataset zurück, aber von welcher Klasse?